storage/mysql/map_storage.go

```diff
@@ -270,32 +271,31 @@ func (m *mapTreeTX) LatestSignedMapRoot(ctx context.Context) (trillian.SignedMap
 
 func (m *mapTreeTX) signedMapRoot(timestamp, mapRevision int64, rootHash, rootSignatureBytes, mapperMetaBytes []byte) (trillian.SignedMapRoot, error) {
```

Comment:
(pre-existing) I think this should return `*trillian.SignedMapRoot`. Protos are usually too large to comfortably pass by value (plus they have a tendency to "fatten" over time).

Maybe add a TODO?

Reply:
Unsure about this.  The entire Log and Map API is expressed in this way. @AlCutter views?

Reply:
I don't really mind either way - the copying the proto is unlikely to be a significant fraction of the operation.

Reply:
FYI, Go style recommends always passing protos by reference